### PR TITLE
PowerBI sample

### PR DESCRIPTION
**Pull request by arturl**

### CODE REVIEW

File: Azure/HelloCloud.Headless/Package.appxmanifest [hunk 1]
```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" xmlns:iot="http://schemas.microsoft.com/appx/manifest/iot/windows10" IgnorableNamespaces="uap mp iot">
+  <Identity Name="HelloCloud-uwp" Publisher="CN=arturl" Version="1.0.0.0" />
+  <mp:PhoneIdentity PhoneProductId="c413c740-710c-4065-b251-9291b6aa05c9" PhonePublisherId="00000000-0000-0000-0000-000000000000" />
+  <Properties>
+    <DisplayName>HelloCloud</DisplayName>
+    <PublisherDisplayName>arturl</PublisherDisplayName>
```
Comment (bfjelds) [inline Azure/HelloCloud.Headless/Package.appxmanifest]: use MSFT for publisher here and in Identity

Comment (bfjelds) [inline Azure/HelloCloud.Headless/Package.appxmanifest]: also copy the pfx file from Blinky and rename it to HelloCloud_TemporaryKey.pfx (that enusres that all of our samples have the same pfx that matches publisher MSFT and have the same expiration date)

File: Azure/WeatherStation.PowerBI/.vs/config/applicationhost.config [hunk 2]
```diff
@@ -0,0 +1,1030 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bfjelds) [inline Azure/WeatherStation.PowerBI/.vs/config/applicationhost.config]: this is a generated file, right?  should it be checked in?

File: Azure/WeatherStation.PowerBI/ApplicationInsights.config [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (bfjelds) [inline Azure/WeatherStation.PowerBI/ApplicationInsights.config]: remove all applicationinsights code/files unless you're actually using it

File: Azure/WeatherStation.PowerBI/Package.appxmanifest [hunk 4]
```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" IgnorableNamespaces="uap mp">
+  <Identity Name="PBIWeather-cc2087bc-f537-4589-93a8-a0a412b73c96" Publisher="Microsoft" Version="1.0.0.0" />
+  <mp:PhoneIdentity PhoneProductId="cc2087bc-f537-4589-93a8-a0a412b73c96" PhonePublisherId="00000000-0000-0000-0000-000000000000" />
+  <Properties>
+    <DisplayName>PBIWeatherApp</DisplayName>
+    <PublisherDisplayName>Microsoft</PublisherDisplayName>
```
Comment (bfjelds) [inline Azure/WeatherStation.PowerBI/Package.appxmanifest]: use publisher=MSFT and the pfx from Blinky

File: Azure/WeatherStation.PowerBI/SensorWeatherDataProvider.cs [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+using Windows.Devices.I2c;
+using Windows.Devices.Enumeration;
+
+namespace WeatherStation
+{
+    class SensorWeatherDataProvider : IWeatherDataProvider
+    {
+        public static async Task<SensorWeatherDataProvider> Create()
+        {
+            String aqs = I2cDevice.GetDeviceSelector("I2C1");
```
Comment (bfjelds) [inline Azure/WeatherStation.PowerBI/SensorWeatherDataProvider.cs:15]: nit: i think the newer APIs are more intuitive:

```
        var i2cController = await I2cController.GetDefaultAsync();
        var sensor = i2cController.GetDevice(new I2cConnectionSettings(0x40));
```

(though i'm not sure it'd work with 10240)

**Comment by arturl** [comment]
Thanks for the feedback, I addressed all comments except I2C initialization suggestion -- I'll use the old API for now, just like other samples do.

**Comment by arturl** [comment]
Thanks!

Artur

**Pull request merged**